cli/aws_ddk/commands/init.py

```diff
@@ -49,12 +55,29 @@ def init_project(name: str, environment: str, template: Optional[str]) -> None:
         },
     )
 
-    # Create virtual environment (.venv)
-    echo(f"Creating virtual environment in `{path}`...")
-    cmd: str = f"{python_exec} -m venv '{path}/.venv'"
-    try:
-        run(cmd)
-    except Exception:
-        secho(f"Failed to run `{cmd}`", blink=True, bold=True)
+    if not generate_only:
+        # Create git repository
+        if not is_in_git_repository(path):
+            echo("Initializing a new git repository...")
+            try:
+                run("git init", path)
+                run("git add .", path)
+                run("git commit --message='Initial commit' --no-gpg-sign", path)
+            except Exception:
+                secho(
+                    "Unable to initialize git repository for your project.",
+                    blink=True,
+                    bold=True,
+                )
+
+            echo("Done.")
+
+        # Create virtual environment (.venv)
+        echo(f"Creating virtual environment in `{path}`...")
+        cmd: str = f"{python_exec} -m venv '{path}/.venv'"
```

Comment:
Can we quickly check what happens if we run this command when there is already a virtual environment with the same name?

Reply:
I meant in case someone runs `ddk init` twice. But if it does not trigger an error then all good